app/views/landing_page/fhir_instructions.html.erb

```diff
@@ -0,0 +1,53 @@
+<section class="hero is-small is-info">
+  <div class="hero-body">
+    <p class="title">
+      <%= link_to root_path, 'aria-label': 'Back to home page' do %>
+        <i class="fa fa-caret-left"></i>
+      <% end %>
+      FHIR Instructions
+    </p>
+  </div>
+</section>
+
+<section class="section">
+  <div class="container">
+    <div class="content is-medium">
+      <p class="title">FHIR Operation</p>
+      <p>
+        This server supports the
+        <%= link_to 'https://spec.smarthealth.cards/#via-fhir-health-cards-issue-operation' do %>
+          FHIR <code>$health-cards-issue</code> operation.
+        <% end %>
+      </p>
+      <p>
+        To retrieve a health card via this FHIR operation, POST a
+        <%= link_to 'Parameters resource', 'https://www.hl7.org/fhir/parameters.html' %> to
+        <code><%= root_url %>Patient/PATIENT_ID/$health-cards-issue</code>
```

Comment:
Just to make this idiot proof, add something saying to replace PATIENT_ID with the id of the patient?